### DISCUSSION TITLE
Roadmap 2022

### DISCUSSION THREAD

**Discussion by coldwarrl** [akkadotnet/akka.net#5439] [General]
Hi,

are there some insights available about the high level roadmap 2022 ? Is it reasonable to say that 1.5 might get released in 2022 and 2.0 in 2023 ?

Thanks!

**Comment by Aaronontheweb** [comment]
@coldwarrl we need to provide an update on this, for sure. We'll put together a contributor meeting for January to discuss it.

But yes, I think those are reasonable assessments. We wanted to get started on 1.5 this year but we reprioritized. Petabridge is in the midst of hiring and expanding its team with some new members due to start in January 2022.

**Reply by Aaronontheweb** [reply]
FYI, we covered this in our two most recent standups: https://github.com/akkadotnet/akka.net/discussions/5691

I'm going to get the entire 1.5 project setup in the Akka.NET org here soon, but you can see the outline on the agenda from our March 2022 meeting.

**Reply by coldwarrl** [reply]
Yes, have watched the land standup. Especially the presentation about the production planning usecase was very interesting :)
